site/src/pages/docs/advanced-streaming-backpressure.mdx

```diff
@@ -31,14 +26,13 @@ Server.builder()
       .service("/big_file.dat", bigFile.asService())
       .build();
 ```
-The <type://HttpFile> loads chunks one by one and send them to the client with backpressure.
 
-## Sending a streaming response using <type://HttpResponseWriter>
+## Sending a streaming response using HttpResponseWriter
 
-If we want to send large data from other than files (e.g. database) with backpressure, we need to implement
-the backpressure by ourselves. Let's start it by implementing the simplified version of <type://HttpFile>.
+To send large data other than files such as database, you need to implement back pressure yourself. Let's start off with implementing a minimal <type://HttpFile>.
+
+Prepare to send a streaming response with <type://HttpResponseWriter> and <type://HttpResponse#streaming()>.
```

Comment:
I wanted to say that `HttpResponseWriter` is returned when you call `HttpResponse.streaming()` which is:
`HttpResponseWriter writer = HttpResponse.streaming();`
So how about:
```suggestion
Prepare to send a streaming response with <type://HttpResponseWriter> returned from <type://HttpResponse#streaming()>.
```

Reply:
returned by?

Reply:
Oh, that's correct. 😄